### PR TITLE
Daily PR March 22

### PR DESCRIPTION
**Pull request by pe7r**
Work with edit functionality.

### CODE REVIEW

File: src/components/EditedForm.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './ToDoItem.css'
+
+class EditedForm extends React.Component {
```
Comment (ruslan-palahin) [inline src/components/EditedForm.js]: Name is not really readable. This should be something like "InlineEditForm"

File: src/components/ItemForm.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './ToDoItem.css'
+
+class ItemForm extends React.Component {
+    render() {
+        return (
+            <div className="todo-item">
```
Comment (ruslan-palahin) [inline src/components/ItemForm.js]: Name is not really readable. This should be something like "ItemView"

File: src/components/ToEditForm.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './ToDoItem.css'
+
+class ToEditForm extends React.Component {
```
Comment (ruslan-palahin) [inline src/components/ToEditForm.js]: Name is not really readable. I've lost tracking what it is... Let's talk later))

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

File: src/components/EditedForm.js (hunk 1 above)
Comment (pe7r) [reply src/components/EditedForm.js]: Renamed. Check please.

File: src/components/ItemForm.js (hunk 2 above)
Comment (pe7r) [reply src/components/ItemForm.js]: Renamed. Check please.

File: src/components/ToEditForm.js (hunk 3 above)
Comment (pe7r) [reply src/components/ToEditForm.js]: Renamed. Check please.

**Pull request merged**